packages/router/router.ts

```diff
@@ -2535,7 +2550,7 @@ function getMatchesToLoad(
     ? Object.values(pendingError)[0]
     : pendingActionData
     ? Object.values(pendingActionData)[0]
-    : null;
+    : undefined;
```

Comment:
Changed to undefined to ensure it properly reflects "no action data" as compared to an `action` that returned `null`